core/src/main/java/gov/nasa/jpl/mudrod/driver/ESDriver.java

```diff
@@ -236,6 +238,7 @@ public String searchByQuery(String index, String type, String query) throws IOEx
     return searchByQuery(index, type, query, false);
   }
 
+  @SuppressWarnings("unchecked")
   public String searchByQuery(String index, String type, String query, Boolean bDetail) throws IOException, InterruptedException, ExecutionException {
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Refactor this method to reduce its Cognitive Complexity from 23 to the 15 allowed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS3776)